hrjobcontract/js/src/job-contract/controllers/form/form-leave.js

```diff
@@ -0,0 +1,38 @@
+define([
+    'job-contract/vendor/fraction',
+    'job-contract/controllers/controllers'
+], function (Fraction, controllers) {
+    'use strict';
+
+    controllers.controller('FormLeaveCtrl',['$scope', '$rootScope', '$filter', '$log', '$q', 'UtilsService',
+      function ($scope, $rootScope, $filter, $log, $q, UtilsService) {
+        $log.debug('Controller: FormLeaveCtrl');
+
+        var promiseNumberOfPublicHolidays = UtilsService.getNumberOfPublicHolidaysInCurrentPeriod();
+
+        promiseNumberOfPublicHolidays.then(function(result){
+          var number = parseFloat(result);
+          if(isNaN(number)) {
+            number = 0;
+          }
+          $scope.numberOfPublicHolidays = number;
```

Comment:
Since you're gonna use the `controller as` syntax (see comment [here](https://github.com/civicrm/civihr/pull/1003/files#diff-4b270671b8363b7cd64d3be53cb39614R563)), you can put every `$scope` property/method in the controller object itself, see for example [here](https://github.com/civicrm/civihr/blob/develop/uk.co.compucorp.civicrm.appraisals/js/src/appraisals/controllers/appraisal-cycle-ctrl.js#L14)